start-kind.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
Could we use `#!/usr/bin/env bash` instead?

Reply:
Done here as well.